app/models/mediaflux_time.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class MediafluxTime
+  def convert(xml_snip:)
+    # takes an xml snip of a nokigiri based time object from a mediaflux response and returns a Princeton time value
```

Comment:
I *love* these code comments. How would you feel about putting them in YARD format so they'll show up nicely in our auto-built documentation? I think if we can get in the habit of doing that it'll make this project more sustainable long-term. 
Here is an example of a YARD method header:
```
# Converts the object into textual markup given a specific format.
#
# @param format [Symbol] the format type, `:text` or `:html`
# @return [String] the object converted into the expected format.
def to_format(format = :html)
  # format the object
end
```

Reply:
Will do! ☝🏿